src/plugins/wizard/server/saved_objects/wizard_app.ts

```diff
@@ -16,7 +16,7 @@ export const wizardSavedObjectType: SavedObjectsType = {
   hidden: false,
   namespaceType: 'single',
   management: {
-    icon: 'visVisualBuilder', // TODO: Need a custom icon here
+    // icon: '', // TODO: Need a custom icon here - unfortunately a custom SVG won't work without changes to the SavedObjectsManagement plugin
```

Comment:
What do we have if we we dont pass this prop?

Reply:
The default icon (see bottom 2):
<img width="1219" alt="Screen Shot 2022-07-21 at 10 46 30 PM" src="https://user-images.githubusercontent.com/1679762/180371509-c7ef05a6-dce5-432d-99dc-2e900f997158.png">


Alternatively, we could just use the visualization icon (other rows), because on this list, there's no distinction between visualization types.